mongoid_lazy_migration.gemspec

```diff
@@ -13,9 +13,12 @@ Gem::Specification.new do |s|
   s.summary     = "Mongoid lazy migration toolkit"
   s.description = "Migrate your documents lazily in atomic, or locked fashion to avoid downtime"
 
-  s.add_dependency("mongoid", "5.0.1")
+  s.add_dependency("mongoid")
```

Comment:
@pdawczak 
> Is this desired? :)
Hmmm. Actually, I was thinking that specifying hard versions of gem dependencies (like for `mongoid` with `5.0.1`) will not make the main gem `mongoid_lazy_migration` be able to be used by apps that depend on slightly different versions of `mongoid`. It will be the responsibility of the hosting app (e.g. `Chopin`) to bundle and lock down a specific version of `mongoid` that would integrate both `mongoid` and this gem correctly without breaking the tests. In other words, I am passing this responsibility to the hosting app. Do you think this is a bad idea?